docs/Symfony.md

```diff
@@ -43,10 +43,25 @@ melodiia:
             description: |
                 This is an awesome API. Really.
             version: 1.0.0
+            pagination:
+                max_per_page_attribute: 'FooQueryParameter'
         # Melodiia comes with some form extensions, it makes your life easier
         # but in some case this could break some parts of an existing application
         # or just not act like you expect. That's why you can't disable extensions.
         form_extensions:
             # enabled by default, set false to disable.
             datetime: true
 ```
+
+III. Override Melodiia services
```

Comment:
Note: this is quite obvious when you use Symfony. But well nothing wrong to say it here to me. (some people would say we should not paraphrase Symfony docs)